etc/org.opencastproject.speechtotext.impl.engine.WhisperCppEngine.cfg

```
@@ -54,3 +54,30 @@
 # This is done by Opencast itself before handing over the audio track.
 # Default: true
 #whispercpp.auto-encode=true
+
+
+## Optional settings for Voice Activity Detection (VAD)
+
+# Enable Voice Activity Detection (VAD)
+#whispercpp.vad=false
+
+# VAD model path
+#whispercpp.vad-model=/usr/share/ggml/silero-v5.1.2-ggml.bin
```

This should be changed to following #6794. Filename is the one used by the download script.
```suggestion
#whispercpp.vad-model=/usr/share/whisper.cpp/models/ggml-silero-v5.1.2.bin
```